toolchain/lexer/tokenized_buffer.cpp

```diff
@@ -828,13 +828,14 @@ auto TokenizedBuffer::PrintToken(llvm::raw_ostream& output_stream, Token token,
       output_stream << ", identifier: " << GetIdentifier(token).index_;
       break;
     case TokenKind::IntegerLiteral():
-      output_stream << ", value: `" << GetIntegerLiteral(token) << "`";
+      output_stream << ", value: '" << GetIntegerLiteral(token).getZExtValue()
```

Comment:
`APInt.print(output_stream, /*isSigned=*/false)` seems like the more appropriate call, as getZExtValue() limits to uint64